products/suse-manager.md

```diff
@@ -0,0 +1,83 @@
+---
+title: SUSE Manager
+category: server-app
+tags: suse
+iconSlug: suse
+permalink: /suse-manager
+versionCommand: zypper info SUSE-Manager-Server-release
+releasePolicyLink: https://www.suse.com/lifecycle
+changelogTemplate: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/__RELEASE_CYCLE__/
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: General Support
+
+releases:
+-   releaseCycle: "5.0"
+    releaseDate: 2024-07-12
+    eol: 2026-06-30
+    latest: '5.0.0'
+    latestReleaseDate: 2024-07-12
+
+-   releaseCycle: "4.3"
+    releaseDate: 2022-06-20
+    eol: 2025-06-30
+    latest: '4.3.13'
+    latestReleaseDate: 2024-06-01
```

Comment:
```suggestion
    latestReleaseDate: 2024-07-04
```

https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.3/index.html

Reply:
Ah, now I understand what you mean! 💡

I think the address on the top only refers to the last document render job. Sometimes they might also fix typos or append notes. I assume that the version history represents the appropriate release date, see here: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.3/index.html#_version_revision_history

So, 2024-06-10 for the very latest 4.3.13. What do you think, does this sound plausible? 🙂

Reply:
Oh, right, I missed it. Thanks for pointing this out. I will review the dates accordingly and document this for maintainers.